src/services/AuthService.tsx

```diff
@@ -65,6 +65,7 @@ const Auth = ({ children }: AuthProps) => {
 		ky.create({
 			credentials: 'include',
 			prefixUrl: BACKEND_URL,
+			retry: 0,
```

Comment:
This means lack of retry. Is it for all POST/GET requests ?

Reply:
this is for all requests made using ky library from this specific handler defined in Auth variable. As far as I'm aware all our request for backend use this Auth.

Reply:
But the default settings of `ky` library seemed to be quite reasonable, as retry limit was set to 2. 

Instead of disabling retries completely I'd configure it such way that the retries are limited to a reasonable amount, such as 3. Note that other fix is on a way: https://github.com/yaptide/yaptide/pull/274

The only thing I am worried about is how timeout was being handled. `ky` documentation says something about `options.timeout`:

![image](https://user-images.githubusercontent.com/7374733/224705256-5ed03956-91ac-4630-84ad-cbfca108dd2e.png)

Reply:
any retries for requests triggered by a button press or on intervals are unnecessary. I could make two configurations to leave retries for auth refreshes and server condition requests but I'm unsure if its the cleanest solution.

Reply:
If we ask for task status every second and one of those requests times out I don't see any reason to use the retry system if the new request will be called either way until we get the status COMPLETED.

> The only thing I am worried about is how timeout was being handled. ky documentation says something about options.timeout:

we use default timeout, which is 10000ms